bot/bot.go

```diff
@@ -29,16 +70,31 @@ type Bot struct {
 
 // InitBot method initalizes a bot
 func InitBot() *Bot {
-	userEmailMap := make(map[string]string)
+	userEmailMap := newMapStateManager()
 	violatorUserMap := make(map[string]string)
 	masterList := strings.Split(os.Getenv("MASTER_LIST"), ",")
 	slack := slackPackage.InitSlack()
 	tock := tockPackage.InitTock()
 	messageRepo := messagesPackage.InitMessageRepository()
 
+	// Start the map manager
+	go userEmailMap.startMapStateManager()
```

Comment:
there''s already a `go` inside of the `startMapStateManager`, you don't need to do it here too.